main.md

```diff
@@ -719,8 +722,7 @@ authentication scheme of `DPoP`. The syntax of the `Authorization`
 header field for the `DPoP` scheme
 uses the `token68` syntax defined in Section 2.1 of [@!RFC7235] 
 (repeated below for ease of reference) for credentials. 
-The Augmented Backus-Naur Form (ABNF) notation [@!RFC5234] syntax 
-for DPoP authentication scheme credentials is as follows:
+The syntax for DPoP authentication scheme credentials is as follows:
```

Comment:
Why is "Augmented Backus-Naur Form (ABNF) notation [@!RFC5234]" being deleted?  It's more specific than the proposed replacement text.